charts/yuki/values.yaml

```
@@ -138,6 +138,10 @@ resources:
   limits:
     memory: "600Mi"
 
+# Pull credentials for the app image, e.g. when nodes can't natively read the
+# registry (GAR, a private ECR repo, etc.). Empty = rely on node-level pull creds.
+imagePullSecrets: [ ]
```

YAMLlint (1.37.1)

[error] too many spaces inside empty brackets

(brackets)